x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/jwt/JwtDateClaimValidator.java

```diff
@@ -62,15 +63,22 @@ public void validate(SignedJWT jwt) {
         switch (relationship) {
             case BEFORE_NOW:
                 if (false == claimInstant.isBefore(now.plusSeconds(allowedClockSkewSeconds))) {
-                    throw new ElasticsearchSecurityException("date claim [" + claimName + "] must be before now", RestStatus.BAD_REQUEST);
+                    throw new ElasticsearchSecurityException(
+                        "date claim [" + claimName + "] must be before now [" + now.toEpochMilli() + "]",
+                        RestStatus.BAD_REQUEST
```

Comment:
Is there anything that logs the claim value ?  It seems like it should be part of this message, but maybe it's logged elsewhere.

Reply:
Thanks. I added the claim value to the message.